megamolbart/megamolbart/inference_perceiver.py

```diff
@@ -0,0 +1,619 @@
+#!/usr/bin/env python3
+
+import logging
+import concurrent.futures
+from functools import partial
+from typing import List
+
+from rdkit import Chem
+from rdkit.Chem import PandasTools, CanonSmiles
+
+import torch
+import pandas as pd
+
+import sys
+import numpy as np
+from collections import defaultdict
+import random
+import logging
+from functools import partial
+from typing import List
+import concurrent.futures
+
+from nemo_chem.models.megamolbart.megatron_bart_model import MegaMolBARTModel
+from nemo_chem.models.megamolbart.megatron_bart_latent_model import MegaMolBARTLatentModel
+
+from nemo.collections.nlp.modules.common.megatron.megatron_init import initialize_model_parallel_for_nemo
+
+initialize_model_parallel_for_nemo(
+    world_size=1,
+    global_rank=0,
+    local_rank=0,
+    tensor_model_parallel_size=1,
+    seed=1234,
+)
+logger = logging.getLogger(__name__)
+
+
+class MegaMolBART():
```

Comment:
```
class MegaMolBART(object):
```
Subtle but important difference in some cases.
Also can you add class description in docstring?

Reply:
Class will be deprecated in new update so will add descriptions in stable future branch